test/playback-watcher.test.js

```diff
@@ -42,6 +42,97 @@ QUnit.module('PlaybackWatcher', {
   }
 });
 
+QUnit.test('skips over gap at beginning of stream if player is paused', function(assert) {
```

Comment:
Minor, but we may want to rename this to something like "skips over gap at beginning of stream if played before content is buffered" as it might reflect the case a bit better.

Reply:
Agree.